strawberry_django/filters.py

```diff
@@ -118,9 +118,24 @@ def build_filter_kwargs(filters):
     return filter_kwargs, filter_methods
 
 
-def apply(filters, queryset, pk=UNSET):
-    if pk is not UNSET:
-        queryset = queryset.filter(pk=pk)
+def apply(
+    filters,
+    queryset,
+    lookup_key=UNSET,
+    lookup_key_django_name=UNSET,
+    lookup_key_value=UNSET,
+):
+    if lookup_key is UNSET:
+        # When using apply() directly
+        lookup_key = "pk"
+
+    if lookup_key_django_name is UNSET:
+        # Default to using the same field name as externally, but allow
+        # the internal name to be overridden.
+        lookup_key_django_name = lookup_key
```

Comment:
Is this to allow using `id` as the GraphQL argument, but map it to `pk` in the Django filters?